static/js/hotkey.js

```diff
@@ -198,6 +198,8 @@ function process_hotkey(e) {
         } else if ($("#subscription_overlay").is(":visible")) {
             subs.close();
             return true;
+        } else if ($('#markdown-help').hasClass('in')) {
+            return false;
```

Comment:
Re: "Invite more users", that one's a little different, in the escape doesn't close it, but should.  Feel free to add an additional commit making that one conform as well. 

Thanks for working on this!